test/functional/p2p_rebroadcast.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+# Copyright (c) 2009-2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+"""Test node rebroadcast logic.
+
+We start by creating a set of transactions to set the rebroadcast minimum fee
+cache to a medium fee rate value.
+
+We then create three sets of transactions:
+    1. aged high fee rate
+    2. aged low fee rate
+    3. recent high fee rate
+
+We add a new connection, trigger the rebroadcast functionality by mining an
+empty block, check that the aged high fee rate transactions were succesfully
+rebroadcast, and that the other two sets were not rebroadcast.
+"""
+
+from test_framework.p2p import P2PTxInvStore
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_approx,
+    assert_greater_than,
+    create_confirmed_utxos,
+)
+import time
```

Comment:
Import standard libraries first, then local modules (this is the opposite order from c++)

Reply:
(this is recommended in PEP8: https://www.python.org/dev/peps/pep-0008/#imports)